addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
That's the same reason we don't do
```js
const {Object} = Ember
```
Even if there are no unforseen side-effects, I think it's misleading for things like `Object`, `String`, `Number` to be overwritten within a module. You probably wouldn't want a variable called `window` for a similar reason.